bugwarrior/services/jira.py

```diff
@@ -174,10 +237,21 @@ def to_taskwarrior(self):
             self.SUBTASKS: self.get_subtasks(),
         }
 
+        extrafields = self.get_extrafields()
+
+        return {**fixed_fields, **extrafields}
```

Comment:
Yeah I agree and have fixed it.

The reason for this format was because I originally wanted to implement this for fields of the form `customfield_12345`, so I called the variable `customfields`. Then I realised I can target _any_ field, so switched to the more generic `extrafields`. But of course now that no longer makes sense!